### ISSUE TITLE
Suggest to run `dashmate setup`

### ISSUE THREAD

**Issue by shumkov** [dashpay/platform#792]
## Expected Behavior

When you run any commands except `setup` with a fresh setup, dashmate should recommend to run `setup` command first. 

## Current Behavior

Since new configs are created with `base` config as default we can see many different errors depend on a command you run.

## Possible Solution

- Initialize dashmate configuration with [default config as null](https://github.com/dashpay/platform/blob/61254f5316a3204b2f45746419d1b8b0f7d01105/packages/dashmate/src/config/systemConfigs/createSystemConfigsFactory.js#L24)
- Recommend to to run the setup command in error message when default group or default config is not set:

```
Error: Default config is not set. Please use '--config' option or set default config
```

**Comment by pshenmic** [comment]
Related https://github.com/dashpay/platform/issues/487

**Comment by shumkov** [comment]
Fixed in v0.24
